addons/website/static/src/components/website_loader/website_loader.xml

```diff
@@ -9,7 +9,10 @@
         <p class="o_website_loader_tip w-25">
             <t t-if="state.showTips">
                 TIP: Once loaded, follow the
-                <span class="o_tooltip o_tooltip_visible bottom o_animated position-relative"></span>
+                <span class="o_tour_pointer o_bottom position-absolute ms-2"
+                    style="--TourPointer__height: 1em; --TourPointer__width: 1em;">
```

Comment:
Move this "weird" style inside the scss file, we already have something targetting its parent `o_website_loader_tip`.
Also, you removed the `o_tooltip` which was attached to some style, you need to remove that style :+1: